apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx

```diff
@@ -428,6 +428,7 @@ const ApplicationView = ({
     folderId: getFolderIdFromEntityId(application.name),
     type: EntityType.Application,
   };
+  const applicationName = application.name.split('__')[0];
```

Comment:
I think it's not correct. What if application was published through api? We need to make validation here that the last part is version and only in this case we should remove the last part. Also name could have __ and here it won't be shown